#!/usr/bin/env python
PACKAGE = "vision"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

# Params for bucket
gen.add("loH", int_t, 0, "Low H bound for bucket", 110, 0, 255)
gen.add("loS", int_t, 0, "Low S bound for bucket", 0, 0, 255)
gen.add("loV", int_t, 0, "Low V bound for bucket", 0, 0, 255)
gen.add("hiH", int_t, 0, "High H bound for bucket", 137, 0, 255)
gen.add("hiS", int_t, 0, "High S bound for bucket", 255, 0, 255)
gen.add("hiV", int_t, 0, "High V bound for bucket", 255, 0, 255)
gen.add("area_thresh", int_t, 0, "Minimum area for bucket", 10000, 1000, 300000)

exit(gen.generate(PACKAGE, "bucketdetector", "bucket"))
